<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inchingWidget</class>
 <widget class="QWidget" name="inchingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>inchingWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="controlBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QPushButton" name="startBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/blue_starticon.png</normaloff>
            <normalon>:/icons/blue_stopicon.png</normalon>:/icons/blue_starticon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="downBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/down-arrow.png</normaloff>:/icons/down-arrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="upBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/up-arrow.png</normaloff>:/icons/up-arrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="leftBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/left-arrow.png</normaloff>:/icons/left-arrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="rightBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/right-arrow.png</normaloff>:/icons/right-arrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="UDcheckBox">
          <property name="text">
           <string>U/D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="velSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="velSpinBox">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>deg/s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statusBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="axisXLabel">
          <property name="text">
           <string>AxisX:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="axisZLabel">
          <property name="text">
           <string>AxisZ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_2">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_2">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>velSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>velSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>velSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
